user-tests/README.md

```diff
@@ -0,0 +1,19 @@
+# User testing information
+
+## Test support
+
+Resources not tied to any single test round.
+
+- [Outreach and recruitment email template](0-support/outreach-email-template.md)
+
+## Test 1: Navigation
+
+August 2022. How do users navigate and access different levels of structure within HTML outputs of Jupyter notebooks?
+
+- [Test script](1-navigation/test-script.md)
+- [Test notebook ()
```

Comment:
Add note describing this notebook! Makes it more approachable and clear!

Reply:
[Done in this commit](https://github.com/Iota-School/notebooks-for-all/pull/26/commits/4a88aa4077394951d9e9f54107473ebb8a3334a9). Thank you for the feedback @smythp! And let me know if this needs more context!